### DISCUSSION TITLE
How to resolve node_modules when they are different between users?

### DISCUSSION THREAD

**Discussion by eliw00d** [yarnpkg/berry#3994] [Q&A]
With yarn, we used to rm -rf node_modules when we had to diagnose issues. But that is no longer recommended. Is there a command we can use to remove and reinstall the node_modules so they are consistent between users? For some reason, one user has a node_modules folder in a workspace but others do not, with the same lockfile.

**Comment by arcanis** [comment]
Do you use the node_modules linker, or PnP? You can still remove the n_m folders manually and run an install, but that shouldn't be necessary(hence why it's not recommended). If it is (and you can consistently reproduce it), feel free to open a bug.

**Reply by eliw00d** [reply]
We use the node_modules linker due to lack of support for PnP in the react native ecosystem. Ah, so just not recommended but might need to be used if we have issues? I'll see if we can get consistent reproduction.
